src/lib/ans-104.ts

```diff
@@ -410,7 +410,10 @@ if (!isMainThread) {
     });
   };
 
-  const filter = createFilter(JSON.parse(workerData.dataItemIndexFilterString));
+  const filter = createFilter(
+    JSON.parse(workerData.dataItemIndexFilterString),
+    log,
```

Comment:
Do we need a child logger here as well?

Reply:
Good catch, I felt it was in the class, it's quite a global filter, I'm setting it to

```ts
log.child({ function: 'ans104IndexFilter' })
```

but any idea for context object could be good or better than this

I'd say Index filter because this is only used once as indexing filter

![2025-02-19 17 35 13](https://github.com/user-attachments/assets/1047ba74-0260-42d5-b514-d93fa3e312a4)